### ISSUE TITLE
Onboarding does not give an error if a non-valid email address is given

### ISSUE THREAD

**Issue by Xelaph** [henrygd/beszel#24]
I tried to give a non-functional email address for the onboarding as I don't think I need that on an internal network. The page gave no errors at all that that was the problem. Would be nice to have some sort of error

**Comment by henrygd** [comment]
Do mean the email address format was valid but it isn't actually in use? For example `nonexistent@gmail.com`?

If so I don't think that's a big issue as admins can change emails for users. However it might be a good idea to allow non-admin users to change their email.

**Comment by Xelaph** [comment]
No, I used a non-valid format. Just a word because I didn't want to even use it as I just use it inside my network and don't have the need for email notifications.

I tried to reproduce it and now I see there is a message, it just gets immediately obscured by Firefox's password manager like this.
![image](https://github.com/user-attachments/assets/3408885c-74b0-4cd1-acc7-21d01fdb6beb)

**Comment by henrygd** [comment]
What the...

Firefox is drawing focus to the email input when you try to submit the form because it's not a valid email. This is the native Firefox validation because it's a `type="email"` field with a `required` attribute. It's not even getting to my custom Valibot validation.

If you try to submit without entering anything in the password field, it correctly displays "Please enter an email address." But putting anything in the password field throws it off. Happens on the regular login form as well.

This seems like a Firefox error but I'll double check that there's nothing funky with the form that could be throwing it off.

**Comment by pixitha** [comment]
This actually also fails if you try to use the +label feature from Gmail. It gives an rather generic error about a password, which quickly disappears.

`user.name@gmail.com` works, but `user.name+beszel@gmail.com` does not. So the email "validation" is more strict than it should be.

**Comment by henrygd** [comment]
@pixitha This is because we split the email at `@` and take the first value to set as the username. PocketBase apparently rejects usernames with `+` in them.

I'll fix it in the next version by using a random value. Or just removing the field since we don't use it for anything.

**Comment by henrygd** [comment]
This is resolved in the next release.
